### PR TITLE
[Tests-Only]And tests for Step class

### PR DESCRIPTION
**Pull request by jasson99**
## Description:
This Pr adds unit tests for Step.

## Related Issue:
- https://github.com/owncloud/owncloud-test-middleware/issues/15

### CODE REVIEW

File: src/gherkin/step.test.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+/* eslint no-undef: 0 */
+const { Step } = require("./index");
+
+const stepData = [
+    [
+        {
+            data:
+                {
+                    tkn: "Given",
```
Comment (dpakach) [inline src/gherkin/step.test.js:9]: more readable
```suggestion
                    token: "Given",
```

**Review by dpakach — COMMENTED** [review]
All looks good, try adding few more invalid steps as well as long and complex steps with lots of data, not only string but numbers as well

File: src/gherkin/step.test.js (hunk 1 above)
Comment (jasson99) [reply src/gherkin/step.test.js]: fixed

**Review by dpakach — APPROVED** [review]
LGTM :+1:

**Pull request merged**